docs/design/Network Topology Aware Scheduling.md

```diff
@@ -544,26 +559,59 @@ type NetworkTopologySpec struct {
     // +optional
     HighestTierAllowed *int `json:"highestTierAllowed,omitempty" protobuf:"bytes,2,opt,name=highestTierAllowed"`
 }
+```
+Based on the above scenario, a single task within the volcano job also requires network topology-aware scheduling.To implement this requirement, modifications to the definitions of the volcano job and podgroup are necessary.
+
+vcjob.spec.task:
+
+```go
+type TaskSpec struct {
+    ...
+    // NetworkTopology defines the NetworkTopology config, this field works in conjunction with network topology feature and hyperNode CRD.
+    // +optional
+    NetworkTopology *NetworkTopologySpec `json:"networkTopology,omitempty" protobuf:"bytes,9,opt,name=networkTopology"`
+}
+```
+
+podgroup
+
+```go
+// PodGroupSpec represents the template of a pod group.
+type PodGroupSpec struct {
+    ...
+    // TaskNetworkTopology defines the NetworkTopology config for each task, this field works in conjunction with network topology feature and hyperNode CRD.
+	  // +optional
+	  TaskNetworkTopology map[string]*NetworkTopologySpec
+}
+
 ```
 
 auto generate code for job, hyperJob, hyperNode,  
 refer the API repo [volcano-sh/apis: The API (CRD) of Volcano (github.com)](https://github.com/volcano-sh/apis) and add CRDs, generate codes and submit to the repo.
 
+
+**Impact on TaskOrderFn:**
```

Comment:
Here should be a best practice that task with network topology should set a higher priority that normal tasks.